File: c/2-control structures/practice/csw-4.c
#include <stdio.h>

int main() {
    int x, x1;     
    int y, y1;     
    int z, z1;    
    scanf("%d %d", &x, &x1);
    scanf("%d %d", &y, &y1);
    scanf("%d %d", &z, &z1);
    if (x < 0 || x1 < 0 || 
        y < 0 || y1 < 0 || 
        z < 0 || z1 < 0) {
        printf("Invalid input\n");
    } else {
        int totalCost = (x * x1) + 
                        (y * y1) + 
                        (z * z1);
        printf("%d\n", totalCost);
    }

    return 0;
}

File: python/03-conditional statements/workout/csw-4.py

a=int(input())
b=int(input())
c=int(input())


if (a < b and b < c) or (c < b and b < a):
    middle = b
elif (b < a and a < c) or (c < a and a < b):
    middle = a
else:
    middle = c


print( middle)